src/core/core.scale.js

```diff
@@ -1206,9 +1206,19 @@ export default class Scale extends Element {
       const color = optsAtIndex.color;
       const strokeColor = optsAtIndex.textStrokeColor;
       const strokeWidth = optsAtIndex.textStrokeWidth;
+      let tickTextAlign = textAlign;
 
       if (isHorizontal) {
         x = pixel;
+
+        if (textAlign === 'left-right') {
```

Comment:
How will this work when labels are rotated on a horizontal axis? It might be better to disable this in that case.

Reply:
This alligment doesn`t affect rotated labels. Add a image based test for this case